### PR TITLE
WIP: g.extension: use git url instead of svn url

### PR DESCRIPTION
**Pull request by AnikaBettge**

### CODE REVIEW

File: scripts/g.extension/g.extension.py [hunk 1]
```diff
@@ -1765,11 +1794,16 @@ def resolve_source_code(url=None, name=None):
     """
     if not url and name:
         module_class = get_module_class_name(name)
-        trac_url = 'https://trac.osgeo.org/grass/browser/grass-addons/' \
-                   'grass{version}/{module_class}/{module_name}?format=zip' \
+        git_url = 'https://github.com/OSGeo/grass-addons/trunk/' \
```
Comment (neteler) [inline scripts/g.extension/g.extension.py]:
```suggestion
        # note: 'trunk' is required to make URL usable for 'svn export' call
        git_url = 'https://github.com/OSGeo/grass-addons/trunk/' \
```

File: scripts/g.extension/g.extension.py [hunk 2]
```diff
@@ -1075,6 +1075,33 @@ def download_source_code_svn(url, name, outdev, directory=None):
...
+    classchar = name.split('.', 1)[0]
+    moduleclass = expand_module_class_name(classchar)
+    if grass.call(['svn', 'export',
+                   url, directory], stdout=outdev) != 0:
+        grass.fatal(_("GRASS Addons <%s> not found") % name)
+    return directory
+
+
 def move_extracted_files(extract_dir, target_dir, files):
     """Fix state of extracted file by moving them to different diretcory
```
Comment (neteler) [inline scripts/g.extension/g.extension.py]:
```suggestion
    """Fix state of extracted files by moving them to different directory
```

**Comment by neteler** [comment]
Tested successfully!

While g.extension might require a "post-SVN" cleanup this workaround avoids that users still fetch addons from the old and frozen SVN-Addon repo rather than https://github.com/OSGeo/grass-addons/

**Comment by neteler** [comment]
ping grass-devs:
the missing merge if this g.extension PR is a showstopper, can anyone please review?

**Comment by landam** [comment]
@neteler backport to relbr76+74 done

**Pull request merged**